CONTRIBUTING.md

```diff
@@ -23,7 +23,7 @@ Before sending your pull requests, make sure you followed this list.
 
 ## Development Environment
 
-The contribution to this project requires setting up a development
+Contribution to this project requires setting up a development
```

Comment:
This isn't precisely a spelling thing, it's more of a grammar thing... Anyway, `The` does the wrong thing here.